### ISSUE TITLE
Mininet Virtual Host

### ISSUE THREAD

**Issue by RoachInUrHost** [nmmapper/python3-nmap#51]
Hi there,
I'm trying to use your library in a virtual host created in Mininet, I ran my Python3 file and it reported "No module named 'nmap3' ", yet your library worked well outside of the host. I wonder if it is available for virtual hosts in Mininet

**Comment by nmmapper** [comment]
Yeah, you have not installed the module. `pip3 install python3-nmap` See if that works.
